app/actions.js

```diff
@@ -52,10 +52,10 @@ const client = new Client({
   }
 })
 // Disable a few slash commands for now
-const removedCommands = ['add', 'channels', 'clear', 'ids', 'names', 'new', 'qr', 'whoami', 'whois']
-removedCommands.forEach((command) => {
-  client.removeCommand(command)
-})
+// const removedCommands = ['add', 'channels', 'clear', 'ids', 'names', 'new', 'qr', 'whoami', 'whois']
+// removedCommands.forEach((command) => {
+//   client.removeCommand(command)
+// })
```

Comment:
should we just remove these lines?

Reply:
maybe?! this was among the few things that didnt work with the latest cabal-client. i didnt spend any time trying to understand why though.

Reply:
ah! maybe that's my fault when i refactored the commands stuff! let's create an issue and link it at the commented out code block?